presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -1373,6 +1374,19 @@ public HiveClientConfig setTemporaryTableCompressionCodec(HiveCompressionCodec t
         return this;
     }
 
+    public boolean getCreateEmptyBucketFilesForTemporaryTable()
```

Comment:
Since it is boolean, let's use `isCreateEmptyBucketFilesForTemporaryTable`? Ditto for the rest of getters.